examples/esp8266/esphub/main.go

```diff
@@ -39,6 +39,14 @@ func main() {
 	adaptor = &dev
 	adaptor.Configure()
 
+	readyled := machine.GPIO{machine.LED}
+	readyled.Configure(machine.GPIOConfig{Mode: machine.GPIO_OUTPUT})
+	readyled.High()
+
+	msgled := machine.GPIO{machine.D7}
```

Comment:
Unfortunately, this is device-specific. Was that intentional?

Reply:
I removed the second LED from this example.